libs/@guardian/react-crossword/src/context/Progress.tsx

```diff
@@ -97,36 +90,11 @@ export const ProgressProvider = ({
 		validator: (progress: unknown) => isValid(progress, { dimensions }),
 	});
 
-	const clearProgress = useCallback(() => {
-		setProgress(getNewProgress(dimensions));
-	}, [dimensions, setProgress]);
-
-	const setCellProgress = useCallback(
-		({ x, y, value }: Coords & { value: string }) => {
-			const newProgress = [...progress];
-
-			if (isUndefined(newProgress[x])) {
-				throw new Error('Invalid x coordinate');
-			}
-
-			if (isUndefined(newProgress[x][y])) {
-				throw new Error('Invalid y coordinate');
-			}
-
-			newProgress[x][y] = value;
-
-			setProgress(newProgress);
-		},
-		[progress, setProgress],
-	);
-
 	return (
 		<ProgressContext.Provider
 			value={{
 				progress,
 				setProgress,
-				setCellProgress,
-				clearProgress,
```

Comment:
why pull these out into their own modules?

Reply:
They now also change the state in the valid answers context